crowd/actions/add_user.py

```diff
@@ -0,0 +1,19 @@
+from lib.base import CrowdBaseAction
+
+class CrowdAddUserAction(CrowdBaseAction):
+	def run(self, username, password, email):
+		# Return True if the user exists in the Crowd application.
+		if not self.status:
```

Comment:
I would move this check on a base action class (either to a ``run`` method on a base class which you overload in each action and call the parent run method or to some other utility method which you call inside each action ``run`` method).

This way you can get rid of the duplicated code. Instead of returning a value, you can just throw an exception inside that method if the authentication fails.